### ISSUE TITLE
Undocumented keyword `INVERSION`

### ISSUE THREAD

**Issue by andreas-el** [equinor/ert#6561]
tldr;
Deprecated keys `USE_EE` & `USE_GE` both lead to `INVERSION`
We allow keywords `IES_INVERSION <int>` and `INVERSION <string>` which map to the same setting
Should be deprecate one of these, and document the other?
None of these are documented at all.
Both variants of inversion settings have been used according to azure logs, although less than 20 incidents over the last 3 months.

-------

FMU-docs does not seem to have anything on this keyword.

```
  def handle_special_key_set(
        self, var_name: str, value: Union[float, int, bool, str]
    ) -> None:
        if var_name in self.DEPRECATED_KEYS:
            logger.warning(
                f"The {var_name} key have been removed" f"use the INVERSION key instead"
            )
        elif var_name == "INVERSION":
            inversion_str_map = {
                "EXACT": 0,
                "SUBSPACE_EXACT_R": 1,
                "SUBSPACE_EE_R": 2,
                "SUBSPACE_RE": 3,
            }
```
Later this is mapped to IES_INVERSION. This setting is also found in the analysis module in the the UI.
Ref.
![Screenshot 2023-11-10 at 14 46 45](https://github.com/equinor/ert/assets/114403625/67b4c727-89db-43a9-ae17-910cf29627bf)

**Comment by andreas-el** [comment]
Also, `IES_INVERSION` is not documented as far as I can tell

**Comment by andreas-el** [comment]
Usage:
`ANALYSIS_SET_VAR STD_ENKF IES_INVERSION 1`

**Comment by andreas-el** [comment]
Using these keys will output that configuration should use `INVERSION` keyword instead
```
    DEPRECATED_KEYS = ["USE_EE", "USE_GE"]
```

**Comment by andreas-el** [comment]
```
ANALYSIS_SET_VAR STD_ENKF INVERSION SUBSPACE_EXACT_R -- ok!
ANALYSIS_SET_VAR STD_ENKF IES_INVERSION SUBSPACE_EXACT_R -- does not work!
ANALYSIS_SET_VAR STD_ENKF INVERSION 3 -- does not work!
ANALYSIS_SET_VAR STD_ENKF IES_INVERSION 3 -- ok!
```

**Comment by andreas-el** [comment]
> Using these keys will output that configuration should use `INVERSION` keyword instead
> 
> ```
>     DEPRECATED_KEYS = ["USE_EE", "USE_GE"]
> ```

This has already been removed.
